### PR TITLE
SafeAreaInsets for android

### PR DESCRIPTION
**Pull request by michaelknoch**
**Type of change:** feature

## Motivation (current vs expected behavior)
Implement SafeAreaInsets for android

## Please check if the PR fulfills these requirements
- [ ] Self-review: I am confident this is the simplest and clearest way to achieve the expected behaviour
- [ ] There are no dependencies on other PRs or I have linked dependencies through Zenhub
- [ ] The commit messages are clean and understandable
- [ ] Tests for the changes have been added (for bug fixes / features)

### CODE REVIEW

File: src/main/java/org/libsdl/app/SDLActivity.kt [hunk 1]
```diff
@@ -82,7 +86,7 @@ open class SDLActivity internal constructor (context: Context?) : RelativeLayout
 
         // Set up the surface
         mSurface = SurfaceView(context)
-        mSurface.setZOrderOnTop(true) // so we can cover the video (fixes Android 8 bug)
+        mSurface.setZOrderMediaOverlay(true) // so we can cover the video (fixes Android 8 bug)
```
Comment (michaelknoch) [inline src/main/java/org/libsdl/app/SDLActivity.kt:89]: setZOrderOnTop
Screenshot 2022-06-17 at 13 42 01

setZOrderMediaOverlay
Screenshot 2022-06-17 at 13 42 44

Comment (michaelknoch) [reply src/main/java/org/libsdl/app/SDLActivity.kt:89]: reference: https://github.com/flowkey/UIKit-cross-platform/pull/353

File: Sources/UIWindow.swift [hunk 2]
```diff
@@ -12,6 +12,7 @@ public class UIWindow: UIView {
     }
 
     open func makeKeyAndVisible() {
+        self.safeAreaInsets = UIWindow.getSafeAreaInsets()
```
Comment (michaelknoch) [inline Sources/UIWindow.swift:15]: we consume this from here
https://github.com/flowkey/NativePlayer/blob/master/FlowkeyPlayer/UIView%2BsafeArea.swift#L14

**Review by ephemer — APPROVED** [review]

**Review by mouselangelo — APPROVED** [review]
🎉

**Review by rikner — APPROVED** [review]

File: Sources/UIWindow+getRootWindowInsets.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (michaelknoch) [inline Sources/UIWindow+getRootWindowInsets.swift:1]: rename file to UIWindow+getSafeAreaInsets

**Pull request merged**